crates/re_viewer/src/ui/welcome_screen/welcome_section.rs

```diff
@@ -40,9 +40,9 @@ pub(super) fn welcome_section_ui(ui: &mut egui::Ui) {
             ui.add_space(4.0);
         };
 
-        bullet_text(ui, "Log with the Rerun SDK in C++, Python, or Rust");
+        bullet_text(ui, "Log data with the Rerun SDK in Python, C++, or Rust");
```

Comment:
I think we should be consistent with the order everywhere, and I believe that order should be lexicographical.

If you strongly disagree, then please change it _everywhere_ in this repository and on landing

Reply:
I'm not sure I agree they need to be consistent everywhere but sure thing. I think they should be ordered as Python, C++, Rust for marketing reasons since that is the order of popularity / usage. I added a commit that does this change to the other places I found it in this repo and here is the pr for landing: https://github.com/rerun-io/landing/pull/782

Reply:
I don't buy the marketing purposes. You eyes take in all the words at once anyway.

You can find more places to change by searching with the regex `C\+\+.*\n?.*Python`

Reply:
Alright, I give up